tests/apm/mod.rs

```diff
@@ -121,10 +124,17 @@ fn logging() {
                 _id: 1 }] }\n",
                &line);
 
+
+    let insert_one_line_start = if v3_3 {
+        "COMMAND.insert_one 127.0.0.1:27017 COMPLETED: { ok: 1, n: 1 } ("
+    } else {
+        "COMMAND.insert_one 127.0.0.1:27017 COMPLETED: { n: 1, ok: 1 } ("
```

Comment:
argh, drives me crazy. this is the expected apm behavior for drivers, yeah? to print it out as ordered by the server, rather than always be lexicographic?